ui/src/shared/components/ClickOutside.tsx

```diff
@@ -1,11 +1,13 @@
 import React, {PureComponent, ReactElement} from 'react'
 import ReactDOM from 'react-dom'
+import {ErrorHandling} from 'src/shared/decorators/errors'
 
 interface Props {
   children: ReactElement<any>
   onClickOutside: () => void
 }
 
+@ErrorHandling
```

Comment:
does it make sense to add this decorator to this? isn't ClickOutside an HOC?

Reply:
OnClickOutside is a HOC but this is a component we use in place of that.

Reply:
I did skip over all the HOCs that were in shared.

Reply:
word